File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/DownloadLanguagesResponseContent.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

import java.util.HashMap;

public class DownloadLanguagesResponseContent extends BaseResponseContent {
    public static final String LOCALIZATION = "localization";


    @SerializedName(LOCALIZATION)
    private HashMap<String, Localize> mLang;//mLocalization

    public HashMap<String, Localize> getLocalization() {
        return mLang;
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/DownloadLanguagesResponseContent.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

class DownloadLanguagesResponseContent : BaseResponseContent() {
    companion object {
        const val LOCALIZATION = "localization"
    }

    @SerializedName(LOCALIZATION)
    private val mLang //mLocalization
            : HashMap<String?, Localize>? = null

    fun getLocalization(): HashMap<String?, Localize>? {
        return mLang
    }
}